content/soarian/dstu2/diagnostic/diagnostic-report.md

```diff
@@ -0,0 +1,94 @@
+
+**CAUTION:** The detail that is in the narrative may not be fully expressed in the structured portion of the FHIR response: The https://www.hl7.org/fhir/narrative-definitions.html#Narrative.status entry in the narrative is valued as "Additional" indicating that more information is provided in the narrative than is provided in the structure. Therefore, if the Application is using the structured entries for any purpose, care must be taken to ensure the clinical content/context of the narrative is preserved.  
+
+## Search
+
+### Parameters
+
+No DiagnosticReport resource specific search parameters are supported. Date / Date range is an optional search parameter.
+
+|Name |Required? | Type | Description
+| --- | --- | --- | --- |
+| patientId | Y | reference | The patient identifier provided in a pre-requisite authorization step. |
+| date | N | [date](http://hl7.org/fhir/DSTU2/search.html#date) as adjusted per implementation notes | Null or specific date or a date range. |
+
+Implementation notes.
+
+While the FHIR DSTU date type is defined, the API relies on the HAPI-FHIR development framework which has certain date behaviors that the developer should be aware of during implementation.  Since these differ in some ways from DSTU2, they are elaborated in detail [here] [date-Param]
+
+[date-Param]: ../../#special-information-regarding-date-parameters "date-param"  
+
+### Header
+To successfully GET Condition, the following headers must be provided.  
+
+    Accept-Type: application/json+fhir
+    Authorization: <OAuth2 Bearer Token>
+
+### Example   
+The following provides an example request and response.
+
+### Request   
+
+GET https://(Base-URL)/DiagnosticReport?patientId=Value&_format=json
+
+where (Base-URL)is provided during registration and patientID value is provided as part of pre-requisite authorization interaction
+
+GET https://fhir-myrecord.sandboxcerner.com/dstu2/d075cf8b-3261-481d-97e5-ba6c48d3b41f/DiagnosticReport?patientId=4342008&_format=json
+
+With optional date parameter: 
+
+GET https://fhir-myrecord.sandboxcerner.com/dstu2/d075cf8b-3261-481d-97e5-ba6c48d3b41f/DiagnosticReport?patientId=4342008&date=\[\>YYYYMMDD]&_format=json.  
+
+###Response
+
+* The API returns qualifying results dated within the implicit or explicitly defined date query parameter.  For response time, response may be limited to 1000 records.  For any non-null response, the API will provide the informational message including the record dates included in the response.  This informational message may be augmented with further detail if noted below.    
+
+* If the query date parameter is null in the query, the API will return qualifying records with dates less than or equal to the date of query as limited by the lesser of 1000 records or 30 days inclusive of the query date.  
```

Comment:
All these date comments belong in the description of the parameter above, or (since there's a lot to digest here), in a _notes_ section right under parameters. EG: http://fhir.cerner.com/millennium/dstu2/individuals/patient/#parameters